docs/io/unified_table.rst

```diff
@@ -0,0 +1,293 @@
+                  ascii.rdb    Yes    .rdb  :class:`~astropy.io.ascii.Rdb`: Tab-separated with a type definition header line
+                  ascii.rst    Yes    .rst  :class:`~astropy.io.ascii.RST`: reStructuredText simple format table
+           ascii.sextractor     No          :class:`~astropy.io.ascii.SExtractor`: SExtractor format table
+                  ascii.tab    Yes          :class:`~astropy.io.ascii.Tab`: Basic table with tab-separated values
+                       fits    Yes    auto  :mod:`~astropy.io.fits`: Flexible Image Transport System file
+                       hdf5    Yes    auto  |HDF5|: Hierarchical Data Format binary file
+                   jsviewer    Yes          JavaScript viewer format (write-only)
+                 pandas.csv    Yes          :func:`pandas.read_csv` and :meth:`pandas.DataFrame.to_csv`
+                 pandas.fwf     No          :func:`pandas.read_fwf` (fixed width format)
+                pandas.html    Yes          :func:`pandas.read_html` and :meth:`pandas.DataFrame.to_html`
+                pandas.json    Yes          :func:`pandas.read_json` and :meth:`pandas.DataFrame.to_json`
+                    parquet    Yes    auto  |Parquet|: Apache Parquet binary file
+                    votable    Yes    auto  :mod:`~astropy.io.votable`: Table format used by Virtual Observatory (VO) initiative
+===========================  =====  ======  ============================================================================================
+
+Details
+-------
+
+.. _table_serialization_methods:
+
+Table Serialization Methods
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``astropy`` supports fine-grained control of the way to write out (serialize)
+the columns in a Table. For instance, if you are writing an ISO format
+Time column to an ECSV ASCII table file, you may want to write this as a pair
+of JD1/JD2 floating point values for full resolution (perfect "round-trip"),
+or as a formatted ISO date string so that the values are easily readable by
+your other applications.
+
+The default method for serialization depends on the format (FITS, ECSV, HDF5).
+For instance HDF5 is a binary format and so it would make sense to store a Time
+object as JD1/JD2, while ECSV is a flat ASCII format and commonly you
+would want to see the date in the same format as the Time object. The defaults
+also reflect an attempt to minimize compatibility issues between ``astropy``
+versions. For instance, it was possible to write Time columns to ECSV as
+formatted strings in a version prior to the ability to write as JD1/JD2
+pairs, so the current default for ECSV is to write as formatted strings.
+
+The two classes which have configurable serialization methods are `~astropy.time.Time`
+and `~astropy.table.MaskedColumn`. See the sections on Time
+:ref:`unified_table_fits_details` and :ref:`unified_table_fits_masked_columns`,
+respectively, for additional information. The defaults for each format are listed below:
```

Comment:
"for additional information how this is handled in fits files"? The links leads to the fits section, while from the text here I expected it to be general for all formats. Instead, som general info is in the next few sentences, but I don't think we have (and maybe also don't need) a section with more details for all formats.